tools/lsp/preview/drop_location.rs

```diff
@@ -1125,10 +1125,9 @@ pub fn create_move_element_workspace_edit(
             let (path, selection_offset) = element.path_and_offset();
             return Some((edit, DropData { selection_offset, path }));
         } else {
-            let children = drop_info.target_element_node.children();
+            let children = &drop_info.target_element_node.children();
             let child_index = {
-                let tmp =
-                    children.iter().position(|c| c == element).expect("We have the same parent");
+                let tmp = children.iter().position(|c| c == element).unwrap_or(usize::MAX - 1);
                 if tmp == children.len() {
```

Comment:
But this can never happen, can it?  `position` will never return the iterator size.